packages/cli-kit/src/public/node/ui.tsx

```diff
@@ -242,6 +242,7 @@ export function renderFatalError(error: Fatal, {renderOptions}: RenderFatalError
 export interface RenderSelectPromptOptions<T> extends Omit<SelectPromptProps<T>, 'onSubmit'> {
   isConfirmationPrompt?: boolean
   renderOptions?: RenderOptions
+  groupOrder?: string[]
```

Comment:
You are not using the `groupOrder` in any `RenderSelectPrompt` function no? I think we don't need this one here

Reply:
Ah I see, is because both this components use the `SelectInput` component, and that's the one that actually uses the groupOrder, we can leave it then, but better move this to `SelectPromptProps`